packages/runner/src/lib/SecScan.ts

```diff
@@ -0,0 +1,50 @@
+import { Reporter } from '@secbox/reporter';
+import {
+  ScanFactory,
+  ScanSettingsOptions,
+  Severity,
+  TargetOptions
+} from '@secbox/scan';
+
+export class SecScan {
+  private _threshold = Severity.LOW;
+  private _timeout: number | undefined;
+
+  constructor(
+    private readonly settings: Omit<ScanSettingsOptions, 'target'>,
+    private readonly scanFactory: ScanFactory,
+    private readonly reporter: Reporter
+  ) {}
+
+  public async run(target: TargetOptions): Promise<void> {
+    const scan = await this.scanFactory.createScan({
+      ...this.settings,
+      target,
+      slowEpTimeout: this._timeout
+    });
```

Comment:
It is not `slowEpTimeout `) You should consider it as `run()` timeout

```suggestion
    const scan = await this.scanFactory.createScan({
      ...this.settings,
      target
    }, {
      timeout: this._timeout
    });
```